content/manuals/compose/how-tos/oci-artifact.md

```diff
@@ -0,0 +1,136 @@
+```
+---
+
+## Publishing your Compose application as an OCI artifact
+
+To distribute your Compose application as an OCI artifact, you can **publish** it to an OCI-compliant registry. 
+This allows others to deploy your application directly from the registry.
+
+The publish function supports most of the composition capabilities of Compose, like overrides, extends or include, [with some limitations](#limitations-and-considerations)
+
+### Steps
+
+1. Navigate to Your Compose Application Directory  
+   Ensure you're in the directory containing your `docker-compose.yml` file or that you are specifying your Compose file with the `-f` flag.
+
+2. Log in to Docker Hub
+   Before publishing, make sure you're authenticated with Docker Hub:
+
+   ```bash
+   $ docker login
+   ```
+
+3. Publish the Compose application to Docker Hub  
+   Use the `docker compose publish` command to push your application as an OCI artifact:
+
+   ```bash
+   $ docker compose publish username/my-compose-app:latest
+   ```
+   or passing multiple Compose files
+   ```bash
+   $ docker compose -f docker-compose.yml -f docker-compose.override.yml publish username/my-compose-app:latest
+   ```
+When publishing you can use options to specify the OCI version, whether to resolve image digests and if you want to include environment variables: 
+* `--oci-version`: Specify the OCI version (default is automatically determined).
+* `--resolve-image-digests`: Pin image tags to digests.
+* `--with-env`: Include environment variables in the published OCI artifact.
+
+Compose checks for you if there isn't any sensitive data in your configuration and displays your environment variables to confirm you want to publish them.
+
+```bash
+...
+you are about to publish sensitive data within your OCI artifact.
+please double check that you are not leaking sensitive data
+AWS Client ID
+"services.serviceA.environment.AWS_ACCESS_KEY_ID": xxxxxxxxxx
+AWS Secret Key
+"services.serviceA.environment.AWS_SECRET_ACCESS_KEY": aws"xxxx/xxxx+xxxx+"
+Github authentication
+"GITHUB_TOKEN": ghp_xxxxxxxxxx
+JSON Web Token
+"": xxxxxxx.xxxxxxxx.xxxxxxxx
+Private Key
+"": -----BEGIN DSA PRIVATE KEY-----
+xxxxx
+-----END DSA PRIVATE KEY-----
+Are you ok to publish these sensitive data? [y/N]:y
+
+you are about to publish environment variables within your OCI artifact.
+please double check that you are not leaking sensitive data
+Service/Config  serviceA
+FOO=bar
+Service/Config  serviceB
+FOO=bar
+QUIX=
+BAR=baz
+Are you ok to publish these environment variables? [y/N]: 
+```
+
+If you refuse the publish process will stop without sending anything to the registry.
+
+---
+
+## Limitations and considerations
+
+There is limitations to publishing Compose applications as OCI artifacts:
+* You can't publish Compose configuration using `include` of local files
```

Comment:
maybe explain: "if you include another compose model, you should publish this one as well"